src/modules/common/containers/LanguageNotAvailableContainer.js

```diff
@@ -2,10 +2,10 @@
 
 import LanguageNotAvailablePage from '../components/LanguageNotAvailablePage'
 import withTheme from '../../theme/hocs/withTheme'
-import compose from 'lodash/fp/compose'
```

Comment:
What do you think about restricting this import? https://eslint.org/docs/rules/no-restricted-imports

Would create  a task for this :)

Reply:
I think it would not be wrong, but we should also restrict the `compose` of `lodash`.

Reply:
I'll create a task.